editor.planx.uk/src/@planx/components/Notice/Editor.tsx

```diff
@@ -105,6 +106,13 @@ const NoticeEditor: React.FC<NoticeEditorProps> = (props) => {
         }}
         value={props.value.notes}
       />
+      <ComponentTagSelect
+        onChange={(value) => props.onChange({
+          ...props.value,
+          tags: value,
+        })}
+        value={props.value.tags}
+      />
```

Comment:
There's a few components where a like-for-like swap to use `ModalFooter` isn't simple. 

Generally this is because either formik is not used (as in this component), or Formik is used via a context wrapper (see `Pay`) instead of the `useFormik()` hook.

A refactor is worth doing I think, but is outside the scope of this PR.